src/components/PlayerUI.tsx

```diff
@@ -13,11 +16,13 @@ function PlayerUI(props: IPlayerUIProps) {
     } = props;
     //Displaying to the user props
     return(
-        <div> 
-            <div>Name: {trackName}</div>
-            <div>Album: {albumName}</div>
-            <div>Artist: {artistName}</div>
-        </div>
+        <AppBar position='fixed' color='primary' sx={{ left: '0', width:'200px', maxheight:'100%',height:'100%' }}>
+            <Toolbar sx={{ flexDirection:"column" }}>
+                <Container  sx={{padding:'20px', paddingTop:'40px'}}><Typography variant='h5' align='center' sx={{color:'rgb(0, 0, 0)'}}>Name: </Typography><Typography align='center' sx={{color:'rgb(0, 0, 0)'}}>{trackName}</Typography></Container>
+                <Container sx={{padding:'20px'}}><Typography variant='h5' align='center' sx={{color:'rgb(0, 0, 0)'}}>Album: </Typography><Typography align='center' sx={{color:'rgb(0, 0, 0)'}}>{albumName}</Typography></Container>
+                <Container sx={{padding:'20px'}}><Typography variant='h5' align='center' sx={{color:'rgb(0, 0, 0)'}}>Artist: </Typography><Typography align='center' sx={{color:'rgb(0, 0, 0)'}}>{artistName}</Typography></Container>
```

Comment:
Instead of using inline styles with the `sx` attribute, can we use css classes? This will allow us to remove some of the clutter in our components and to reduce repeated code